### ISSUE TITLE
Dynamically adjust the distance_to_tracker filter to the displayed radar range

### ISSUE THREAD

**Issue by Nickduino** [Springvar/home-assistant-flightradar24-card#9]
Sorry if the title is confusing, I'd like the list of aircrafts to display all aircrafts shown on the radar **even after zooming in/out with the scroll wheel**, would this be possible?

**Comment by Springvar** [comment]
v0.0.11-pre.2 has added the possibility to use ${radar_range} as a filter value for the radar.

Let me know if this let you configure the list the way you want.

**Comment by Springvar** [comment]
Added a new example for this in the README documentation. Also released a new prerelease version v0.0.11-pre.3 with a flag to defer update of the list on mobile devices.

The example is simply:
```yaml
type: custom:flightradar24-card
filter:
  - field: distance_to_tracker
    comparator: lte
    value: ${radar_range}
```

Selected flights will still be listed even if they become out of range.

**Comment by Nickduino** [comment]
> ```yaml
> distance_to_tracker
> ```

If only I had read the title of my issue in which I clearly mention `distance_to_tracker`...


Well, it works, thanks!
I've had a few hiccups but maybe that was me playing with a range way too big. I'll let you know if things start to happen again.
